survey/sas-ad-computer-prefix-export.ps1

```
@@ -0,0 +1,145 @@
+            -ComputerName '' `
+            -ADStatus 'ad_query_failed' `
+            -ADProbeMethod 'active_directory_module_prefix' `
+            -Notes $_.Exception.Message
+    }
+}
+elseif ($hasDsquery) {
+    try {
+        $raw = & dsquery.exe computer -name $likePattern 2>&1
+        if ($LASTEXITCODE -eq 0 -and $raw) {
+            foreach ($line in @($raw | Where-Object { $_ -and $_.ToString().Trim() })) {
+                $host = ''
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'host' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)